sql/src/main/java/org/apache/druid/sql/calcite/planner/DruidOperatorTable.java

```diff
@@ -268,8 +268,8 @@ public class DruidOperatorTable implements SqlOperatorTable
                    .add(new MultiValueStringOperatorConversions.Append())
                    .add(new MultiValueStringOperatorConversions.Prepend())
                    .add(new MultiValueStringOperatorConversions.Concat())
-                   .add(new MultiValueStringOperatorConversions.Contains())
-                   .add(new MultiValueStringOperatorConversions.Overlap())
+                   .add(MultiValueStringOperatorConversions.CONTAINS)
```

Comment:
If we did it would be sort of like Calcite's `SqlStdOperatorTable`, which has a ton of statics in it. It's definitely useful when writing rules. That's the reason I changed this one— I needed to use it in a rule.